packages/content/src/enrichments.ts

```diff
@@ -573,20 +576,47 @@ export const enrichContent = (
   const defaults: Partial<IHubContent> = {
     errors: [],
   };
-  const contentWithPortalUrls = {
+  const contentWithDefaults = {
     ...defaults,
     ...content,
-    ...getPortalUrls(content, requestOptions),
   };
   // fetch any missing or requested enrichments
-  return fetchEnrichments(contentWithPortalUrls, requestOptions).then(
+  return fetchEnrichments(contentWithDefaults, requestOptions).then(
     (fetched) => {
-      // enrich dates now that we have metadata (if any)
-      const enriched = _enrichDates(fetched);
-      // if this is a layer (type: Feature Layer, Table, Raster Layer)
-      // or a feature or map service and caller has supplied a layer id
-      // we want the content to represent the layer instead of the service
-      return getLayerContent(enriched, requestOptions.layerId) || enriched;
+      const { layerId } = requestOptions;
+      const {
+        item,
+        slug,
+        errors,
+        data,
+        metadata,
+        groupIds,
+        orgId,
+        // TODO: org, ownerUser
+        server,
+        layers,
+        // TODO: these should be fetched instead of pulled from the dataset
```

Comment:
Wait, fetched as in fetched from the item? Because it seems like we are fetching these values from somewhere...

Reply:
in `fetchContent()` I'm going to have to add enrichments for these that fetch them from the Hub API.

Reply:
Ah, got it. I was confused because this line is part of destructuring an object called `fetched`